.env.example

```diff
@@ -4,9 +4,9 @@
 # env variables required for running the demo
 
 # mandatory: URL of the container registry to use.
-# Default (local Kind registry): localhost:5000
+# Default (local Kind registry): 127.0.0.1:5000
 # Azure Container Registry (ACR) example: osmci.azurecr.io/osm
-export CTR_REGISTRY=localhost:5000
+export CTR_REGISTRY=127.0.0.1:5000
```

Comment:
Sean and I were running into odd kind registry networking issues when communicating to localhost vs 127.0.0.1
I switched this over, and I do not see any issues hopefully :)

Reply:
Did you figure out why localhost doesn't work?

Reply:
No clue, still debugging. It works fine with any other application, though.

Reply:
Since this issue is very specific to your env and this is an example file, you could override it without needing to change the default here.

Reply:
It's entirely possible 127.0.0.1 is an objectively better default, but we won't know that for sure until we figure out why localhost isn't working.

Reply:
Regardless, this change seems unrelated to this PR. I'd make this and other similar changes in a separate PR so they can be reviewed in isolation.

Reply:
> running into odd kind registry networking issues when communicating to localhost vs 127.0.0.1
I switched this over

@schristoff This led me to think this is an issue specific to your environment (Kind cluster). Are you suggesting localhost doesn't work on arm64? If yes, I'd add this to the commit and PR description, and also a comment in the example file to not use `localhost` as we'd want to keep the demo as generic as possible.

Reply:
I think I've now narrowed down the issue (with the help of @nojnhuh ) - your original suggestion is correct @shashankram  - I will open up another PR with the appropriate changes. :)